Updated TAP installation flow with  env and comments to make it more readable (#2744)
pivotal/docs-tap: install-online/profile.hbs.md

1. Set up environment variables for installation use by running:

    ```console
    # Set tanzunet as the source registry to copy the TAP packages from
    export IMGPKG_REGISTRY_HOSTNAME_0=registry.tanzu.vmware.com
    export IMGPKG_REGISTRY_USERNAME_0=MY-TANZUNET-USERNAME
    export IMGPKG_REGISTRY_PASSWORD_0=MY-TANZUNET-PASSWORD

    # User’s registry where TAP packages will be copied to
    export IMGPKG_REGISTRY_HOSTNAME_1=MY-REGISTRY
    export IMGPKG_REGISTRY_USERNAME_1=MY-REGISTRY-USER
    export IMGPKG_REGISTRY_PASSWORD_1=MY-REGISTRY-PASSWORD
    # Note: Above IMGPKG_* envs are being used by imgpkg command only

    # Registry to fetch TAP Packages from
    export INSTALL_REGISTRY_USERNAME=”${IMPKG_REGISTRY_USERNAME_1}”
    export INSTALL_REGISTRY_PASSWORD=”${IMPKG_REGISTRY_PASSWORD_1}”
    export INSTALL_REGISTRY_HOSTNAME=”${IMPKG_REGISTRY_HOSTNAME_1}”
    export TAP_VERSION=VERSION-NUMBER
    export INSTALL_REPO=TARGET-REPOSITORY

>     # User’s registry where TAP will store built images and as well as the tanzu build -service dependencies. Ensure these credentials have write permission.

Reviewer: ```suggestion
    # User’s registry where TAP will store built images and as well as the tanzu build-service dependencies. Ensure these credentials have write permission.
```

    imgpkg copy -b registry.tanzu.vmware.com/tanzu-application-platform/tap-packages:${TAP_VERSION} --to-repo ${INSTALL_REGISTRY_HOSTNAME}/${INSTALL_REPO}/tap-packages
    ```

<details>

<summary>If you do not want to relocate images</summary>

> [Relocate images to a registry](#relocate-images) is strongly recommended but not required for installation. If you skip this step, you can use the following values to replace the corresponding variables:

Reviewer: ```suggestion
[Relocate images to a registry](#relocate-images) is strongly recommended but not required for installation. If you skip that step, you can execute the following commands in your command line to set the source of the OCI Images to be TanzuNet:
```

```console
# Registry to fetch TAP Packages from.
export INSTALL_REGISTRY_USERNAME=TANZUNET_REGISTRY_USERNAME
export INSTALL_REGISTRY_PASSWORD=TANZUNET_REGISTRY_PASSWORD
export INSTALL_REGISTRY_HOSTNAME=”registry.tanzu.vmware.com”
export TAP_VERSION=VERSION-NUMBER
export INSTALL_REPO="tanzu-application-platform"

> # User’s registry where TAP will store built images and as well as the tanzu build -service dependencies. Ensure these credentials have write permission.

Reviewer: ```suggestion
# User’s registry where TAP will store built images and the Tanzu build-service dependencies. Ensure these credentials have write permission.
```
